tensorflow/tools/api/golden/v1/tensorflow.pbtxt

```diff
@@ -1284,6 +1284,14 @@ tf_module {
     name: "function"
     argspec: "args=[\'func\', \'input_signature\', \'autograph\', \'experimental_autograph_options\'], varargs=None, keywords=None, defaults=[\'None\', \'None\', \'True\', \'None\'], "
   }
+  member_method {
+    name: "fused_batch_norm_grad_v3"
```

Comment:
You need an apidef file hiding the new ops, like the ones for the existing ops https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/python_api/api_def_FusedBatchNormGradV2.pbtxt

Reply:
Thanks for pointing out this.

I've added the missing two .pbtxt files for the V3 op.